### ISSUE TITLE
Eliminate relative imports

### ISSUE THREAD

**Issue by slifty** [TechAndCheck/tech-and-check-alerts#307]
We have a lot of relative imports (`../../../../utils/blah`) which is sloppy and hard to parse.

There are a few tactics that would allow us to always write paths in terms of root.  Lets talk through them and consider picking one.

**Comment by reefdog** [comment]
Yes please. [Here's an excellent round-up of various options.](https://gist.github.com/branneman/8048520) My criteria:

1. The use syntax should be standard.
2. We should (obviously) still be able to do ad hoc relative imports.
3. Developers should not have to do anything new to start using it.
4. Since this is non-standard behavior, the method we choose should stand out to Node developers who aren't familiar with our codebase. (This eliminates the "silent root" methods like options 4-5.)
5. Third-party libraries should be minimal, and those used should be actively maintained.

With that in mind, I like their option 0, ["The Alias"](https://gist.github.com/branneman/8048520#0-the-alias), the best. My recommendation is we commit to just a single root alias, named `$root` (not `@root`; see [this issue](https://github.com/ilearnio/module-alias/issues/27)), so we can `import { cleaner } from '$root/utils/cnn'`, which is both legible and immediately recognizable as non-standard. I also recommend we use the [`addAlias()` registration method](https://www.npmjs.com/package/module-alias#advanced-usage) rather than package.json configuration, to further handcuff ourselves to a single root alias.

We do have to add a package, but it's actively-maintained with no worrisome issues. [We will also have to add a bit of Jest configuration.](https://github.com/ilearnio/module-alias/issues/46#issuecomment-546154015)
